src/server.rs

```diff
@@ -227,6 +235,15 @@ fn read<'r>(
     // against FxA and the method, and either returns OK or an error. We need to reraise it to the
     // handler.
     let request_id = headers.headers.get("request_id");
+
+    let mut tags = Tags::default();
+    tags.extra.insert("user_id".to_owned(), user_id.clone());
+    tags.extra.insert("device_id".to_owned(), device_id.clone());
+    tags.extra
+        .insert("request_id".to_owned(), format!("{:?}", request_id));
```

Comment:
minor nit I'm fine with ignoring, this looks like a String already

```suggestion
        .insert("request_id".to_owned(), request_id);
```

Reply:
It probably is. I tend to overkill for reporting stuff like this just to make sure that some value is captured. Funny that clippy didn't pick it up.